articles/140_topic_and_service_name_mapping.md

```diff
@@ -279,31 +349,10 @@ Services are governed by the same algorithm, but in some implementations may req
 In the specific case of RTI Connext's Request-Reply implementation, they append the `Request` and `Reply` strings to the topic names.
 Therefore, it would be safest to assume the Service name limit to be less 8 more characters.
 
-## Compare and Contrast with ROS 1
-
-In order to support a mapping to the more restrictive DDS topic name rules, these rules are in some ways more constraining than the rules for ROS 1.
-Other changes have been proposed for convenience or to remove a point of confusion that existed in ROS 1.
-In ROS 2, topic and service names differ from ROS 1 in that they:
-
-- must separate the tilde (`~`) from the rest of the name with a forward slash (`/`)
-
-  - This is done to avoid inconsistency with how `~foo` works in filesystem paths versus when used in a ROS name.
-
-- may contain substitutions which are delimited with balanced curly braces (`{}`)
+#### Substitution of the Namespace Delimiter
```

Comment:
could you move this subsection higher up in the "alternatives" section so that the "ROS to DDS Name Conversion Examples" section has context